src/third_party/CMakeLists.txt

```diff
@@ -59,6 +59,16 @@ add_library(lz4
   )
 target_include_directories(lz4 INTERFACE .)
 
+#---------------------------------------------------------------------------------------------------
+# zstd
+#---------------------------------------------------------------------------------------------------
+file(GLOB_RECURSE ZSTDSRC "zstd/*.c")
```

Comment:
Please avoid globbing, and be verbose instead (sorry, but it's a thing for me to be 100% sure that we only build & include what's intended, and not accidentally build files that happen to be on someones drive).